tests/test_data_cleaning.py

```diff
@@ -28,26 +28,34 @@ def test_insert_rollout_handles_groups(frame):
     """
     If grouping columns are given to insert_rollout, a separate rollout is inserted for each group.
     """
-    frame = frame.with_columns(date=pl.col("date").str.strptime(pl.Date, "%Y-%m-%d"))
+    frame = frame.with_columns(
+        time_end=pl.col("time_end").str.strptime(pl.Date, "%Y-%m-%d")
+    )
     rollout = [dt.date(2020, 1, 1), dt.date(2021, 1, 1)]
-    group_cols = ("geography",)
+    group_cols = [
+        "geography",
+    ]
     frame = iup.CumulativeUptakeData(frame.drop("indicator"))
 
     output = frame.insert_rollout(rollout, group_cols)
 
     assert output.shape[0] == 7
     assert (
-        output["date"].value_counts().filter(pl.col("date") == rollout[0])["count"][0]
+        output["time_end"]
+        .value_counts()
+        .filter(pl.col("time_end") == rollout[0])["count"][0]
```

Comment:
I think another way to do this logic is to filter and look at the number of rows:
```python
output['time_end'].filter(pl.col("time_end") == rollout[0]).shape[0] == 2
```